karma.conf.js

```diff
@@ -40,7 +40,7 @@ module.exports = (config) => {
       // 'test/TiledPixiTrackTests.js',
       // 'test/ViewManipulationTests.js',
       // 'test/ChromSizesTests.js',
-      // 'test/ViewportProjectionTests.js',
+      'test/ViewportProjectionTests.js',
```

Comment:
```suggestion
      // 'test/ViewportProjectionTests.js',
```

Should be commented out when being merged.